packages/code-du-travail-frontend/src/outils/DureePreavisLicenciement/steps/InfosCCn.js

```diff
@@ -0,0 +1,19 @@
+import { StepInfoCCnMandatory } from "../../common/InfosCCn";
+import data from "@cdt/data...preavis-licenciement/data.json";
+
+import { isNotYetProcessed } from "../../common/situations.utils";
+
+StepInfoCCnMandatory.validate = values => {
+  const errors = {};
+  const { ccn } = values;
+  if (Object.keys(values).length === 0) {
```

Comment:
I'm not usre this is used this InfoCCMandatory handle mandatory field

Reply:
Done